runtime/src/main/java/org/corfudb/runtime/object/transactions/ReadSetEntry.java

```diff
@@ -5,6 +5,7 @@
 /** An entry in a transaction read set.
  * Created by mwei on 12/19/16.
  */
+@Deprecated
 public class ReadSetEntry {
```

Comment:
This isn't really a publicly exposed-api, so why should it be marked deprecated? Just remove the class and any references if you're going to deprecate it.